### PR TITLE
Updated the README to give smaller step by step instructions.

### PR DESCRIPTION
**Pull request by EECOLOR**
Note that I removed the alternate 'go install' options to reduce confusion.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,35 +9,42 @@ Prerequisites
 - [Go](https://golang.org/).
 - The [GOPATH environment variable](https://github.com/golang/go/wiki/GOPATH) correctly set.
 
+An example setup:
+
+```
+export GOROOT="$HOME/opt/go1.4.2"
```
Comment (paulp) [inline README.md:22]: You're not supposed to set GOROOT directly unless you are doing something outside the defaults with your go installation. See (and possibly link to) https://github.com/golang/go/wiki/InstallTroubleshooting

Comment (paulp) [inline README.md:22]: You can be explicit about the small number of platforms we know anything about:

```
brew install go # OSX
dpkg -i golang # ubuntu, they'll figure it out for other linuxes
```

File: README.md [hunk 2]
```diff
@@ -46,40 +53,65 @@ Usage: suffuse <options> [path path ...]
...
+
 # These files are effectively indistinguishable from "real" files.
 % ls -l /mnt/seq.txt#1,3p
--rw-r--r--  1 paulp  wheel  6 Jun 30 11:57 /mnt/seq.txt#1,3p
+-rw-r--r--  1 user  user  6 Jun 30 11:57 /mnt/seq.txt#1,3p
+```
+
+_Kill the suffuse instance_
+```
+% kill 9134
```
Comment (paulp) [inline README.md]: kill %1 
Although in your case based on the earlier transcript it's actually %2. Regardless I don't think it's necessary to have a tutorial on using the shell here. We're a while from needing that kind of handholding.

Comment (EECOLOR) [inline README.md]: Haha, are you sure? I'm quite a noob when it comes to the shell. For example: I don't know what `kill %1` does.

I think it's quite clear what `kill 9134` does in light of the `&` output above.

Comment (paulp) [inline README.md]: See where it says `[2] 9134` ? You can kill that with `kill 9134` or with `kill %2` which will almost always be `kill %1` - it's only higher if you already had background processes. When you have to type out pids you're doing it wrong.

**Comment by EECOLOR** [comment]
@paulp please check again

**Comment by paulp** [comment]
I'll merge this for the moment, but in general I think it's a lot better to outsource installation instructions to the relevant project. I'd rather just link to the Go installation page and let them be in charge of how to convey the relevant information.

**Pull request merged**